Jint/Native/JsValue.cs

```diff
@@ -566,6 +590,43 @@ internal virtual JsValue DoClone()
 
         internal virtual bool IsCallable => this is ICallable;
 
+        /// <summary>
+        /// https://tc39.es/ecma262/#sec-ordinaryhasinstance
```

Comment:
I was wondering such a weird name ... well it's the specs

Reply:
Indeed, I had trouble accepting it too 😄